### PR TITLE
[Backend][myAccount] Implement backend APIs for my account page

### PR DESCRIPTION
**Pull request by WangJ509**
Please follow the guidelines in [Contribution.md](https://github.com/social-tw/social-tw-website/blob/main/CONTRIBUTING.md) first, then start to create your pull request here.

## Summary

Implement backend APIs for my account page

## Linked Issue

#198 

## Details

1. Refactor PostService and CommentService, remove epks in fetchPosts and fetchComments
2. Implement myaccount.ts to new APIs
3. Implement `fetchMyAccountPosts`, `fetchMyAccountComments`, `fetchMyAccountVotes`
4. Query parameters for new APIs includes:
- epks: `_` separated string of epoch keys, e.g. `key1_key2_key3`
- sortKey: `publishedAt` | `voteSum`
- direction: `asc` | `desc`

## Impacted Areas

Remove `epks` for the following APIs
- /api/post
- /api/comment

## Tests

Mention all types of tests (unit, integration, performance, etc.) that you've carried out and if any specific environment or configuration is needed to run them.

## Possible Impacts

Describe any existing features that might be affected by this change, or any potential backward compatibility issues, if any.

## Visual Materials

Examples of new APIs
```shell
curl --location 'localhost:8000/api/my-account/posts?epks=111_222_333&direction=asc&sortKey=voteSum'
```

```shell
curl --location 'localhost:8000/api/my-account/comments?epks=111_222_333&direction=asc&sortKey=publishedAt'
```

```shell
curl --location 'localhost:8000/api/my-account/votes?epks=111_222_333&direction=desc&sortKey=publishedAt'
```

## Verification Steps

Provide a step-by-step guide for others to reproduce your results and verify your changes.

## Todo

-   [ ] Unit tests for `fetchMyAccountPosts`, `fetchMyAccountComments`, `fetchMyAccountVotes`

## Checklist

- [x] Define API spec
- [x] Implement API
- [ ] Implement unit tests

### CODE REVIEW

File: packages/relay/src/routes/myAccount.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            direction: 'asc' | 'desc',
+            db: DB,
+        ) => Promise<any[]>,
+    ) => {
+        const epks = req.query.epks as string | undefined
+        const parsedEpks = epks?.split('_') || []
+        if (parsedEpks.length === 0) {
+            throw new InternalError(
+                'epks must be specified and should be a non-empty string',
+                40,
```
Comment (andy78644) [inline packages/relay/src/routes/myAccount.ts:25]: this is 400?

**Review by andy78644 — COMMENTED** [review]
should add the condition of error test

File: packages/relay/src/routes/myAccount.ts (hunk 1 above)
Comment (WangJ509) [reply packages/relay/src/routes/myAccount.ts]: fixed

**Comment by WangJ509** [comment]
@andy78644 I have fixed the typo and added a test for error conditions. Please help me review those changes 😊

**Review by andy78644 — APPROVED** [review]

**Review by andy78644 — APPROVED** [review]

**Pull request merged**